lib/ae_mdw/sync/async_store_server.ex

```diff
@@ -49,6 +54,14 @@ defmodule AeMdw.Sync.AsyncStoreServer do
     {:noreply, state}
   end
 
+  def handle_cast({:write_mutations_without_async, mutations, done_fn}, state) do
+    _new_state = State.commit_db_without_async(State.new(), mutations)
```

Comment:
Once the database transaction is an optimistic one this write operation might be lost/dropped on a conflict with the transaction from the sync server.